fastlane/Fastfile

```diff
@@ -38,20 +39,40 @@ EXTENSIONS_BUNDLE_IDENTIFIERS = %w[
   WordPressThisWeekWidget
   WordPressIntents
 ].map { |suffix| "#{APP_STORE_VERSION_BUNDLE_IDENTIFIER}.#{suffix}" }
-
 ALL_BUNDLE_IDENTIFIERS = [APP_STORE_VERSION_BUNDLE_IDENTIFIER, *EXTENSIONS_BUNDLE_IDENTIFIERS]
-
-require_relative 'lanes/localization-lanes'
-
-# Use this instead of getting values from ENV directly
-# It will throw an error if the requested value is missing
+JETPACK_APP_IDENTIFIER = 'com.automattic.jetpack'
+
+# Environment Variables — used by lanes but also potentially actions
+Dotenv.load(USER_ENV_FILE_PATH)
+Dotenv.load(PROJECT_ENV_FILE_PATH)
+GHHELPER_REPO = 'wordpress-mobile/wordpress-iOS'
+ENV['PROJECT_NAME'] = 'WordPress'
+ENV['PUBLIC_CONFIG_FILE'] = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.Public.xcconfig')
+ENV['INTERNAL_CONFIG_FILE'] = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.internal.xcconfig')
+ENV['DOWNLOAD_METADATA'] = './fastlane/download_metadata.swift'
+ENV['PROJECT_ROOT_FOLDER'] = PROJECT_ROOT_FOLDER + '/'
+ENV['APP_STORE_STRINGS_FILE_NAME'] = 'AppStoreStrings.po'
+ENV['FL_RELEASE_TOOLKIT_DEFAULT_BRANCH'] = 'trunk'
+
+# Use this instead of getting values from ENV directly. It will throw an error if the requested value is missing
 def get_required_env(key)
-  unless ENV.key?(key)
-    UI.user_error!("Environment variable '#{key}' is not set. Have you setup #{USER_ENV_FILE_PATH} correctly?")
-  end
+  UI.user_error!("Environment variable '#{key}' is not set. Have you setup #{USER_ENV_FILE_PATH} correctly?") unless ENV.key?(key)
   ENV[key]
 end
 
+
+########################################################################
+# Imports of other Fastfiles
```

Comment:
Semantics nitpick: I'm unsure about "other Fastfiles" to describe the `lanes/*.rb` files 🤔 

How do you feel about the following?

```suggestion
# Import domain specific lanes
```